.gitignore

```diff
@@ -24,7 +24,7 @@ docs/xml/
 index.md
 
 # test artifacts
-*.log
```

Comment:
I'd leave *.log in there, just in case, or if ever going back to an older commit.  Can't hurt.